quconot/implementations/mct_no_auxiliary_relative.py

```diff
@@ -37,7 +37,7 @@ def verify_mct_cases(
         relative_phase: true / false (D)
         clean_auxiliary: true (D) / false
         wasted_auxiliary: true / false (D)
-        separable_wasted_auxiliary: true / false (D)    `requires wasted_auxiliary set to True`
```

Comment:
this is strange, what happened here?

Reply:
last time you asked to remove the `#`, so I put the ` apostrophe ` to make them readable in the documentation

Reply:
will be fixed in issue #13